clarity_ext/repository/step_repository.py

```diff
@@ -171,5 +174,9 @@ def current_user(self):
         "target_concentration": "Target Concentration",
         "target_volume": "Target Volume",
         "volume": "Current sample volume (ul)"
+    },
+    "ResultFile": {
```

Comment:
It's not even certain that volume and concentration are relevant for Analytes in all steps, but in some steps it's needed. According to teh reasoning above, there should be separate classes, depending on which step the analytes/result files are in. This is a relevant question however, the fact that Analyte and ResultFile will grow very big as they will cover all different kind of steps. The different UDFs only have meaning for a certain step.